pyproject.toml

```diff
@@ -80,6 +80,8 @@ sqlalchemy = { version = "^2.0.18", optional = true }
 getdaft = { version = ">=0.2.12", optional = true }
 cachetools = "^5.5.0"
 pyiceberg-core = { version = "^0.4.0", optional = true }
+thrift-sasl = { version = ">=0.4.3", optional = true }
+kerberos = { version = "1.3.1", optional = true }
```

Comment:
this is causing the CI issue. Im not sure if we really require this dependency

Reply:
Good call, let me check!